index.html

```diff
@@ -51,6 +51,155 @@ <h1>I am Talha.<br>Glad to see you!</h1>
+                <div class="card image"><img src="card-images/4.png" alt=""></div>
+                <div class="card left-block">
+                    <div class="card head">
+                        <h2>Tonic</h2>
+                        <ul class="caption">
+                            <li class="canopy"><h3>CANOPY</h3></li>
+                            <li><img src="Counter.svg" alt="dot"></li>
+                            <li><h3>Back End Dev</h3></li>
+                            <li><img src="Counter.svg" alt="dot"></li>
+                            <li><h3>2015</h3></li>
+                        </ul>
+                    </div>
+                    
+                    <div class="card text">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <ul class="tags">
+                        <li class="tag">HTML</li>
+                        <li class="tag">CSS</li>
+                        <li class="tag">JavaScript</li>
+                    </ul>
+                    <div class="card action"><button class="see-project">See Project</button></div>
+                </div>
+            </div> 
+            
+        </section>
+
+        <section class="about-myself">
+            <h2>About Myself</h2>
+            <p>Hello. I’m a software developer! I can help you build a product, feature or website, look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="lets-connect">LET'S CONNECT</p>
+            <div class="icons">
+                <ul>
+                    <li>
+                        <a href="https://www.github.com/"><img src="icons/icon-github.svg" alt="GitHub">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://www.linkedin.com/"><img src="icons/icon-linkedin.svg" alt="LinkedIn">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://www.angellist.com/"><img src="icons/icon-angellist.svg" alt="angellist">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://www.twitter.com/"><img src="icons/icon-twitter.svg" alt="Twitter">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://www.medium.com/"><img src="icons/icon-medium.svg" alt="Medium">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="card action"><button class="see-project">Get my resume</button></div>
+        <ul class="myself-list">
+            <li class="languages">Languages</li>
+            <li class="frameworks">Frameworks</li>
+            <li class="skills">Skills</li>
```

Comment:
- [ ] Kindly, Make sure that your design looks similar to the design in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
The technology categories should look like this : 
FIGMA DESIGN 👍 
![image](https://user-images.githubusercontent.com/71874425/153680126-ad9cf8ca-a056-49d1-bffb-79c1d992ead7.png)
YOUR DESIGN 👎 

![image](https://user-images.githubusercontent.com/71874425/153680294-8b5d5d5c-a947-484a-80a6-a9d0b2aab29d.png)
- [ ] Please, kindly fix this to meet the Figma design provided